controls/nginx_service.rb

```diff
@@ -0,0 +1,23 @@
+title 'Nginx base plan tests'
+
+plan_name = input('plan_name', value: 'nginx')
+plan_ident = "#{ENV['HAB_ORIGIN']}/#{plan_name}"
+
+control 'base-plans-nginx-service' do
+    impact 1.0
+    title 'Ensure Nginx is running as a service'
+    desc ''
+
+    # Get the ident of the package
+    status_cmd = command("hab svc status #{plan_ident}")
```

Comment:
Might be worth ensuring the status call is successful here e.g. 
```
    describe status_cmd do
        its('exit_status') { should eq 0 }
        its('stdout') { should_not be_empty }
    end
```

Reply:
Have added this :+1: